src/popup/components/buttons/BtnMain.vue

```diff
@@ -85,7 +85,7 @@ export default defineComponent({
   flex: 1 1 0;
   width: fit-content;
   min-height: 40px;
-  padding: 5px 16px;
+  padding: 0 16px;
```

Comment:
The top and bottom padding should not be set to 0. Actually I see that a change was made in figma regarding the button settings. The padding top and bottom should be equal to `8px`, and the `line-height` to `24px` instead of `20`.